decimalSuite_test.go

```diff
@@ -203,6 +203,9 @@ func runTest(context Context64, testVals decValContainer, testValStrings testCas
 	calcRestul := calculatedContainer.calculated
 
 	if calculatedContainer.calculatedString != "" {
+		if testValStrings.testFunc == "compare" && calculatedContainer.calculatedString == "-2" && testVals.expected.IsNaN() {
```

Comment:
Testcases support both decimal and int returns; This is not specified in the standard and will not be followed